packages/core/src/customer/strategies/stripe-upe/stripe-upe-customer-strategy.ts

```diff
@@ -80,14 +80,24 @@ export default class StripeUPECustomerStrategy implements CustomerStrategy {
                 appearance,
             });
 
-            const linkAuthenticationElement = this._stripeElements.getElement(StripeElementType.AUTHENTICATION) || this._stripeElements.create(StripeElementType.AUTHENTICATION);
+            const billingAddress = this._store.getState().billingAddress.getBillingAddress();
+            const options = billingAddress?.email? { defaultValues: { email: billingAddress?.email } } : { } ;
```

Comment:
This seems a bit confusing. 
Looking at code `options` interface is like 
```
export interface StripeElementsCreateOptions {
    fields?: FieldsOptions;
    wallets?: WalletOptions;
    allowedCountries?: any;
}
```

Which seems a bit different from `{defaultValues: {}} ` Just want to check if I am missing anything?

Reply:
Hey @animesh1987, on the last version I have, this interface does have 
 ```
* All available options are here https://stripe.com/docs/js/elements_object/create_payment_element
 */
export interface StripeElementsCreateOptions {
    fields?: FieldsOptions;
    wallets?: WalletOptions;
    allowedCountries?: string[];
    defaultValues?: ShippingDefaultValues | CustomerDefaultValues;
}

interface ShippingDefaultValues {
    name: string;
    address: {
        line1: string;
        line2: string;
        city: string;
        state: string;
        postal_code: string;
        country: string;
    };
}

interface CustomerDefaultValues {
    email: string;
}
```

please let me know if I misunderstood the comment

Reply:
Maybe we can make it eaiser to read in this way.
```suggestion
            const { email } = billingAddress;
            const options = email ? { defaultValues: { email } } : {} ;
```

Reply:
@bc-peng I made the change but with a little difference, "**`email`**" is already destructured in line 44